### PR TITLE
Resolve #1090, #1091, #1099

### LINKED ISSUE #1090: Disable continue in transfer form on getTxCost error
Also show an error snackbar

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/components/EmailTransferFormComponent.jsx [hunk 1]
```diff
@@ -89,6 +91,7 @@ class EmailTransferFormComponent extends Component<Props, State> {
       formError
     } = transferForm
 
+    console.log(error)
```
Comment (InVincible2016) [inline src/components/EmailTransferFormComponent.jsx]: remove

File: src/components/ReviewComponent.jsx [hunk 2]
```diff
@@ -24,7 +24,8 @@ type Props = {
   actionsPending: {
     submitTx: boolean,
     getTxFee: boolean
-  }
+  },
+  error: Object
```
Comment (InVincible2016) [inline src/components/ReviewComponent.jsx]: error is not an object

Comment (timothywangdev) [reply src/components/ReviewComponent.jsx]: fixed

File: src/components/EmailTransferFormComponent.jsx (hunk 1 above)
Comment (timothywangdev) [reply src/components/EmailTransferFormComponent.jsx]: fixed

File: src/wallets/utils.js [hunk 3]
```diff
@@ -123,9 +123,9 @@ async function getTxFee ({
   try {
     gasCost = await getGasCost(txObj)
   } catch (e) {
-    if (e.message && e.message.includes('gas required exceeds allowance (10000000)')) {
-      // 44945 covers contract interaction cost
-      gasCost = await getGasCost(txObj, '44945')
+    if (e.message && e.message.includes('gas required exceeds allowance')) {
+      // 44945 covers contract interaction (createErc20Wallet) cost
```
Comment (InVincible2016) [inline src/wallets/utils.js]: why 133134 ? also update comment

**Comment by InVincible2016** [comment]
Fix repeated commits

**Pull request merged**